Content.Shared/_RMC14/Xenonids/Parasite/VictimInfectedEmoteEvent.cs

```diff
@@ -0,0 +1,6 @@
+using Content.Shared.Chat.Prototypes;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared._RMC14.Xenonids.Parasite;
+
+public record struct VictimInfectedEmoteEvent(ProtoId<EmotePrototype> Emote);
```

Comment:
I missed this on the first review sorry, this needs [ByRefEvent] added and then you can raise it by ref so it doesn't get copied (for performance).